src/main/webapp/script.js

```diff
@@ -143,15 +162,36 @@ function toTooltipString(userComments) {
  * @return {String} HTML format to display its message and magnitude
  */
 function userCommentAsString(comment) {
-  commentMagnitude = comment.magnitude;
+  commentMagnitude = comment.magnitudeScore;
   return comment.commentMsg + '<br> Magnitude Score: ' + commentMagnitude;
 }
 
 /**
- * Convert range to string
- * @param {Range} range a range object.
- * @return {String} a string made from the range.
+ * Convert a Range object to a string
+ * @param {*} range a range object
```

Comment:
Was the specific object type removed because EsLint complained that it didn't know what a Range object was?

You can keep the `*` but maybe update the comment to make it more explicit that `range` should be a representation of a java Range object?